c/driver/postgresql/postgresql_test.cc

```diff
@@ -162,6 +161,53 @@ TEST_F(PostgresConnectionTest, GetInfoMetadata) {
   ASSERT_THAT(seen, ::testing::UnorderedElementsAreArray(info));
 }
 
+TEST_F(PostgresConnectionTest, GetObjectsGetCatalogs) {
+  ASSERT_THAT(AdbcConnectionNew(&connection, &error), IsOkStatus(&error));
+  ASSERT_THAT(AdbcConnectionInit(&connection, &database, &error), IsOkStatus(&error));
+
+  if (!quirks()->supports_get_objects()) {
+    GTEST_SKIP();
+  }
+
+  {
```

Comment:
nit: why the extra nesting?

Reply:
The nesting is done in the parent test that this was copied from, but is superfluous here